pre-cache/check_space

```diff
@@ -8,8 +8,8 @@ TOKEN=$(cat ${SERVICEACCOUNT}/token)
 CACERT=${SERVICEACCOUNT}/ca.crt
 let high=$(curl --cacert ${CACERT} --header "Authorization: Bearer ${TOKEN}" -X GET ${APISERVER}/api/v1/nodes/${NODE_NAME}/proxy/configz | \
             chroot /host jq '.kubeletconfig.imageGCHighThresholdPercent')
-let size=$(df -h /var/lib/ | awk '{ print $2 }' | sed -n '2p' | sed  's/G//')
-let used=$(df -h /var/lib/ | awk '{ print $3 }' | sed -n '2p' | sed  's/G//')
+let size=$(df /var/lib/ | awk '{ print $2 }' | sed -n '2p')
```

Comment:
I propose using `(( expr ))` instead of `let expr` to avoid [SC2219](https://github.com/koalaman/shellcheck/wiki/SC2219), wdyt?
```
(( high=$(curl --cacert ${CACERT} --header "Authorization: Bearer ${TOKEN}" -X GET ${APISERVER}/api/v1/nodes/${NODE_NAME}/proxy/configz | \
               chroot /host jq '.kubeletconfig.imageGCHighThresholdPercent') ))
(( size=$(df /var/lib/ | awk '{ print $2 }' | sed -n '2p') ))
(( used=$(df /var/lib/ | awk '{ print $3 }' | sed -n '2p')))
```